timelock-server/src/suiteTest/java/com/palantir/atlasdb/timelock/MultiNodePaxosTimeLockServerIntegrationTest.java

```diff
@@ -349,10 +351,15 @@ public void canGetAllNamespaces() {
         cluster.waitUntilAllServersOnlineAndReadyToServeNamespaces(
                 ImmutableList.of(cluster.clientForRandomNamespace().namespace()));
 
-        Set<String> namespacesAfterRestart = getKnownNamespaces();
-        assertThat(namespacesAfterRestart).contains(randomNamespace);
-        assertThat(namespacesAfterRestart).doesNotContain("learner", "acceptor");
-        assertThat(Sets.difference(knownNamespaces, namespacesAfterRestart)).isEmpty();
+        Set<String> knownNamespacesAfterRestart = getKnownNamespaces();
+        assertThat(knownNamespacesAfterRestart).contains(randomNamespace);
+        assertThat(knownNamespacesAfterRestart).contains(randomNamespace);
+        assertThat(knownNamespacesAfterRestart).doesNotContain("learner", "acceptor");
+        assertThat(Sets.difference(knownNamespaces, knownNamespacesAfterRestart))
+                .isEmpty();
```

Comment:
nit: I realise this was from the original, but I would prefer writing something like `assertThat(knownNamespacesAfterRestart).isSubsetOf(knownNamespaces);` for better error messages